server_logic.py

```diff
@@ -0,0 +1,32 @@
+from typing import List, Dict
+
+"""
+This file can be a nice home for your move logic, or to write helper functions.
+
+We have started it for you, with a function to help remove the 'neck' direction
+from the list of possible moves!
+"""
+
+def avoid_my_neck(my_head: Dict[str, int], my_body: List[dict], possible_moves: List[str]) -> List[str]:
```

Comment:
Type Annotations! Too much, or a prefect opportunity to demonstrate the awesome that is PEP-484? I could be talked out of this.